threshold_tools_application_examples.ipynb

```diff
@@ -121,9 +124,10 @@
     "For this section, please select:\n",
     "- \"daily\" timescale,\n",
     "- \"2m Air Temperature\",\n",
+    "- \"degC\" for units,\n",
```

Comment:
Do the units matter here? If not, I would remove this from the text. Also, the 2m Air Temperature variable has been renamed to "Air Temperature at 2m", so that should be changed here as well.

Reply:
Ah I am seeing that degC is referenced further down the notebook. Maybe keep the degC then?